public/components/create_report/delivery/delivery.tsx

```diff
@@ -13,60 +13,202 @@
  * permissions and limitations under the License.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import {
-    EuiFieldText,
-    EuiFormRow,
-    EuiPageHeader,
-    EuiTitle,
-    EuiPageContent,
-    EuiPageContentBody,
-    EuiHorizontalRule,
-    EuiTextArea,
-  } from '@elastic/eui';
+  EuiFieldText,
+  EuiFormRow,
+  EuiPageHeader,
+  EuiTitle,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiHorizontalRule,
+  EuiTextArea,
+  EuiCheckbox,
+  EuiSpacer,
+  EuiComboBox,
+  EuiText,
+  EuiLink,
+  EuiPopover,
+  EuiListGroup,
+} from '@elastic/eui';
 import { DeliveryRecipientsBox } from './delivery_recipients_box';
-import { DeliveryChannelRadio } from './delivery_channel_radio';
+import { email_recipient_options } from './delivery_constants';
 
-const ReportDelivery = (props) => {
-  const { 
-    deliveryEmailSubject, 
-    onChangeDeliveryEmailSubject,
-    deliveryEmailBody,
-    onChangeDeliveryEmailBody
-  } = props;
+const insert_placeholder_options = [
+  {
+    label: 'Report details URL',
+    href: '#',
+    iconType: 'link',
+    size: 's',
+  },
+  {
+    label: 'Report source URL',
+    href: '#',
+    iconType: 'link',
+    size: 's',
+  },
+  {
+    label: 'File download URL',
+    href: '#',
+    iconType: 'link',
+    size: 's',
+  },
+  {
+    label: 'Report creation timestamp',
```

Comment:
is this actual a timestamp, 12323434?
if so, would be human friendly convert to local time or UTC time 
```
moment(dateTime).local().format()
moment(dateTime).utc().format()
```

Reply:
This is just a label with onClick action to be added after connecting to the back-end